lib/ecosystems/enrich_cyclonedx.go

```diff
@@ -47,16 +47,20 @@ var cdxEnrichers = []cdxEnricher{
 	enrichCDXSupplier,
 }
 
+var cdxPackageVersionEnrichers = []cdxPackageVersionEnricher{
+	enrichCDXLicense,
+}
+
 func enrichCDXDescription(comp *cdx.Component, data *packages.Package) {
 	if data.Description != nil {
 		comp.Description = *data.Description
 	}
 }
 
-func enrichCDXLicense(comp *cdx.Component, data *packages.Package) {
-	if data.NormalizedLicenses != nil {
-		if len(data.NormalizedLicenses) > 0 {
-			expression := data.NormalizedLicenses[0]
+func enrichCDXLicense(comp *cdx.Component, data *packages.Version) {
+	if data.Licenses != nil {
+		if *data.Licenses != "" {
```

Comment:
this looks a bit like a red flag. don't we have `data.NormalizedLicenses` on the package version response? this change would mean that we are potentially enriching with unsanitized data. e.g. `MIT,Apache-2.0` is not a valid SPDX license expression.

Reply:
looks like the `ecosyste.ms` response for a specific package version doesn't have a normalised licenses field, no

Reply:
I raised an issue with ecosyste.ms about this here: https://github.com/ecosyste-ms/packages/issues/960

Reply:
@paulrosca-snyk could we move forward with this approach, but `strings.Split()` the value and build an SPDX license expression with `strings.Join()`? The current behaviour is to concatenate with an `OR`, but [we have an open issue on this](https://github.com/snyk/parlay/issues/77) and should switch to `AND`.

Reply:
@mcombuechen definitely, but are we sure we want to proceed with the `AND` change, it doesn't seem like it will fix the issue, just change it in the other direction (i.e we'll be reporting `OR` clauses incorrectly as `AND`, but maybe that's better than the current behaviour?)

Reply:
From my understanding they can be either and we just don't get this information back from ecosyste.ms (assuming they even have it stored)